app/scripts/templates/complete_2fa.mustache

```diff
@@ -0,0 +1,18 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-confirm-header">{{#t}}New Sign-in to Firefox{{/t}}</h1>
+    <h1 id="fxa-confirm-signin-header">{{#t}}For added security, please confirm this sign-in to 
+    begin syncing with this device.{{/t}}</h1>
+  </header>
+
+  <section>
+    <form novalidate>
+      <div class="button-row">
+        <button id="submit-btn" type="submit">{{#t}}Confirm{{/t}}</button>
+      </div>
+    </form>
+    <div class="links">
+        If you think someone is trying to gain access to your accound, <a href="/reset_password" class="right reset-password" data-flow-event="forgot-password">{{#t}}please change your password{{/t}}</a>
```

Comment:
Two modifications to correctly handle l10n and to minimize pain for the translators:

1. Because this line contains HTML, the whole line needs to be surrounded with an `{{#unsafeTranslate}}{{/unsafeTranslate}}` and the internal `{{#t}}{{/t}}` needs to be removed.
2. That anchor element contains a bunch of attributes. To avoid having the translators translate all of those, we can use some string interpolation.

Together, those work out to:

```mustache
{{#unsafeTranslate}}If you think someone is trying to gain access to your accound, <a %(escapedResetLinkAttributes)s>please change your password</a>{{/unsafeTranslate}}
```

Then in views/complete_2fa.js->setInitialContext, pass in an `escapedResetLinkAttributes` field:

```js
setInitialContext (context) {
  context.set({
    // No need to escape the string, it contains no user generated input
    escapedResetLinkAttributes: 'href="/reset_password" class="right reset-password" data-flow-event="forgot-password"'
  });
}
```

Reply:
Also, small metrics drive-by, can we change the flow event attribute to:

```
data-flow-event="link.reset-password"
```

I didn't do it initially but going forwards we want to have a `link.` prefix on all these click events, to make their meaning clearer when writing queries. And `forgot-password` is meant to be specific to the "Forgot password?" text we show on the sign-in form, whereas this is resetting the password for a different reason.

Reply:
Also, there is a typo in the text: `accound` => `account`